tripal_bulk_loader/includes/tripal_bulk_loader.loader.inc

```diff
@@ -476,7 +483,7 @@ function tripal_bulk_loader_load_data($nid, $job_id) {
       $loaded_without_errors = FALSE;
       break;
     }
-    tripal_bulk_loader_progress_bar($total_lines, $total_lines);
```

Comment:
Changed because this hid the fact that total_lines included a header while num_lines did not